### ISSUE TITLE
sysUpdater fails if firmware setup by Mac OS X

### ISSUE THREAD

**Issue by cloudburst** [profi200/sysUpdater#20]
If firmware files are copied into the sd:/updates folder using Mac OS X, it will make copies of each .cia file that start with ._ (AppleDouble files) so it will look like this:

```
 -rwxrwxrwx 1 3DS 3DS 21584 0004800F484E4C41.cia
 -rwxrwxrwx 1 3DS 3DS 4096 ._0004800F484E4C41.cia
```

https://en.wikipedia.org/wiki/AppleSingle_and_AppleDouble_formats

This causes sysUpdater to fail at https://github.com/profi200/sysUpdater/blob/master/source/main.cpp#L131
```cpp
void installUpdates(bool downgrade)
{
	std::vector<fs::DirEntry> filesDirs = fs::listDirContents(u"/updates", u".cia;"); // Filter for .cia files
```

with error
```
titleException:
main.cpp:148
Failed to get CIA file info!
```

More robustness checking would be a welcome improvement here.

**Comment by cloudburst** [comment]
Nevermind this was fixed in fed1587863db9faed51b32c52f4e92d96f6362f4, just no one has pulled that into their sysUpdater builds.

**Comment by profi200** [comment]
Yes, it was fixed. Because i don't provide versions with memchunkhax v2 compiled in you are better by asking the people which provide these versions. They just need to pull the changes i made.
